docs/user/hbase/kerberos.rst

```diff
@@ -0,0 +1,34 @@
+                             Kerberos
+========
+
+GeoMesa includes initial support for HBase clusters which are authenticated using Kerberos. Currently, keytabs are supported.
+
+Kerberos functionality should be configured by appending the following properties to ``hbase-site.xml``:
+
+- ``hbase.geomesa.keytab``
+- ``hbase.geomesa.principal``
+
+All applications will require access to ``hbase-site.xml`` on their classpath in order to obtain the correct configuration.
+
+Note that Kerberos support is only available in HBase 1.1.2 or newer.
+
+Development & Testing
+---------------------
+
+GeoMesa Kerberos support was developed against Hortonworks Data Platform 2.6 authenticating against an MIT KDC.
```

Comment:
For your testing on HDP 2.6, do you have a link to the steps you used to Kerberize the cluster?  If so, let'd be great!  I'm hoping to find some time to test this out over the next couple of days.